README.md

```diff
@@ -1,16 +1,21 @@
 
-Samples demonstrating how to use [axe](https://www.deque.com/axe/) for automated accessibility testing of websites in [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) builds.
-
-Automated accessibility tests can detect some common accessibility problems such as missing or invalid properties. But most accessibility problems can only be discovered through manual testing. We recommend [Accessibility Insights](https://accessibilityinsights.io/docs/en/web/overview), a free and open source dev tool that walks you though assessing a website for 100% [WCAG 2.0 AA](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_overview&versions=2.0&levels=aaa) coverage.
+This project contains sample projects (see next section [Available samples](#available-samples)) demonstrating how to implement automated accessibility testing in [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) builds using [axe-core](https://github.com/dequelabs/axe-core), the same accessibility scanning engine used in [Accessibility Insights for Web](https://github.com/microsoft/accessibility-insights-web).
 
 ## Available samples
 
-* [typescript-selenium-webdriver](./typescript-selenium-webdriver/README.md): Uses [TypeScript](https://typescriptlang.org), [Jest](https://jestjs.io/), [Selenium](https://www.seleniumhq.org), and [axe-webdriverjs](https://www.npmjs.com/package/axe-webdriverjs) to test a web page for accessibility issues. Shows how to baseline existing failures with [Jest Snapshot Testing](https://jestjs.io/docs/en/snapshot-testing) and how to visualize failures in [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) using [SARIF](https://sarifweb.azurewebsites.net/).
+The following sample projects specify the main technologies used. A team that uses comparable tools and frameworks should be able to refer to the sample and update their existing test to incorporate automated accessibility checks.
+
+* **[Sample 1: typescript-selenium-webdriver-sample](./typescript-selenium-webdriver/README.md)**: 
+  * Useful for teams using technologies similar to TypeScript, Jest, Selenium, and axe-webdriverjs. 
 
 *Are we missing a sample you'd like to see? [File a sample request](https://github.com/microsoft/axe-pipelines-samples/issues/new?assignees=&labels=sample_request&template=feature_request.md&title=Sample+Request%3A+%3Csample+name+here%3E) or [submit a pull request](./CONTRIBUTING.md)!*
 
+## Disclaimer
+
+Automated accessibility tests can detect some common accessibility problems such as missing or invalid properties. But most accessibility problems can only be discovered through manual testing. We recommend [Accessibility Insights](https://accessibilityinsights.io/docs/en/web/overview), a free and open source dev tool that walks you though assessing a website for 100% [WCAG 2.0 AA](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_overview&versions=2.0&levels=aaa) coverage.

```

Comment:
I wonder if we should make this a little less harsh; we had at least one user that was discouraged to continue at all when they read this. Maybe we can work with PMs on more encouraging wording (replacing "some/most" with "many/many other")? Maybe something like "Automated accessibility tests can detect many common accessibility problems such as color contrast issues, and we recommend including them in your builds. However, there are also many types of accessibility problems that can only be discovered through manual testing. ..."

Reply:
@ferBonnin Thoughts on this?

Reply:
we (Lisa Louise, PM team, Chelsea and some devs ) talked about this a lot. We need to be super crisp that automated checks don't provide coverage for everything. That's why we are blunt and super explicit about it.

Reply:
@ferBonnin I agree with the desire to be crisp and explicit, but if the user study feedback is telling us that the current language discourages users from running automated scans at all, I think it makes sense for us to try to iterate on something that retains most of the explicitness while being a little less discouraging. Maybe a clause between the current first and second sentences adding something to encourage action, eg "It's a great first step" or "They are a good first line of defense for the simplest types of issues" or something?

Reply:
Fer and I discussed further offline; let's keep this wording as is for the sake of this PR, we'll follow up on it separately :)

Reply:
Filed #43 to track separately